### ISSUE TITLE
Extract Associated Probability

### ISSUE THREAD

**Issue by morgan-hd** [jiegzhan/multi-class-text-classification-cnn-rnn#24]
This is awesome code. I am fairly new to TF and have tried to google the answer myself but can't figure it out. How do I also extract the associated probability with the label that it predicts?

**Comment by morgan-hd** [comment]
UPDATED:

```
		with tf.name_scope('output'):
			self.W = tf.Variable(tf.truncated_normal([hidden_unit, num_classes], stddev=0.1), name='W')
			b = tf.Variable(tf.constant(0.1, shape=[num_classes]), name='b')
			l2_loss += tf.nn.l2_loss(W)
			l2_loss += tf.nn.l2_loss(b)
			self.logits = tf.nn.xw_plus_b(output, self.W, b)
			self.probs = tf.nn.softmax(self.logits)
			self.probabilities = tf.argmax(self.probs,1,name='probabilities')
			self.scores = tf.nn.xw_plus_b(output, self.W, b, name='scores')
			self.predictions = tf.argmax(self.scores, 1, name='predictions')
```
